metabase/metabase
# Bookmark Dashboards

This PR includes an improvement to the API from tweaking the `delete` function in entities, suggested by @ranquild, as well as using a `selectors` file to feed props updates.

## How to Test

1. Go to a Dashboard page
2. Open the kebab menu to the right
3. Find the Bookmark item
4. Click to toggle it

![image](https://user-images.githubusercontent.com/380816/158280944-bd724fb5-5cf8-4e46-a916-3da6c33674da.png)

Upon bookmarking a dashboard, when visiting a Collection page, you should see the bookmark on the left sidebar.

![image](https://user-images.githubusercontent.com/380816/158281076-ba99229d-f539-45df-95ce-7f85bf63cb42.png)
